distributions/docker/replication/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:11-jre-slim
+COPY maven/${app.artifactId}-${project.version}.jar /opt/replication/app.jar
+COPY maven/demo-config/keystores /opt/replication/demo-config/keystores
+COPY maven/demo-config/application.properties /opt/replication/demo-config/application.properties
+
+ENV truststore_pass changeit
```

Comment:
If we change the way your application module is packaging your application we can basically negate the need for any of the lines from here down. This would have the dual benefit of making it easier to run both in and out of a docker container